sass/variables-site/_fonts.scss

```
@@ -1,7 +1,7 @@
 // Font and typographic variables
 
-$font__body: Georgia, Garamond, "Times New Roman", serif;
-$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+$font__body: Georgia, Garamond, "Times New Roman", serif !default;
+ $font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif !default;
```

Extra space at the start of this, I assume accidental?
```suggestion
$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif !default;
```

---

Ah shoot! Yes, entirely accidental - it has been removed in a82df1f.